packages/apollos-ui-auth/src/Entry.js

```diff
@@ -66,7 +70,7 @@ const Entry = ({
         Platform.OS === 'android' ? StatusBar.currentHeight : 0
       }
     >
-      <BackgroundComponent />
+      {BackgroundComponent}
```

Comment:
This is technically a breaking API change. Might have to do something like what [`react-native` core](https://github.com/facebook/react-native/blob/master/Libraries/Lists/VirtualizedList.js#L970-L975), would look something like:

```suggestion
      {React.isValidElement(BackgroundComponent) ? (
        BackgroundComponent
      ) : (
        < BackgroundComponent />
      )}
```

Reply:
Nice! 👍